packages/dashmate/src/listr/tasks/startNodeTaskFactory.js

```diff
@@ -28,6 +28,11 @@ function startNodeTaskFactory(
    * @return {Object}
    */
   function startNodeTask(config) {
+    // check core is not reindexing
+    if (config.get('core.reindex', true)) {
+      throw new Error(`Your dashcore node in config [${config.name}] in process of reindex, please finish it first`)
```

Comment:
`Your dashcore node in config [${config.name}] is reindexing, please allow the process to complete first`